src/Jab.FunctionalTests.Common/ContainerTests.cs

```diff
@@ -1272,6 +1272,21 @@ public void SupportsNamedServices()
         [Singleton(typeof(IAnotherService), typeof(AnotherServiceImplementation), Name="OnlyNamed")]
         [Singleton(typeof(ServiceImplementationWithNamed<IService>))]
         internal partial class SupportsNamedServicesContainer
+		{
```

Comment:
Something is wrong with the formatting here. 

Do you mind replacing these tabs with spaces?